notify_send.py

```diff
@@ -160,6 +160,10 @@
         '/usr/share/icons/hicolor/32x32/apps/weechat.png',
         'Path to an icon to be shown in notifications.'
     ),
+    'desktop-entry': (
+        'weechat',
+        'The desktop id for the weechat instance'
```

Comment:
All descriptions of the other options end with a period (`.`), so I suggest to be consistent and end also this one with a period.